cms/templates/js/course-outline.underscore

```diff
@@ -78,7 +78,7 @@ if (xblockInfo.get('graded')) {
                         </a>
                     </li>
                 <% } %>
-                <% if (xblockInfo.canBeDeleted()) { %>
+                <% if (xblockInfo.showSectionContent()) { %>
```

Comment:
Can you explain this change as I'm concerned it might affect other things in Studio? Did we previously add .canBeDeleted() in an earlier PR?